frontend/app/services/subscription-service.server.ts

```diff
@@ -111,6 +117,35 @@ function createSubscriptionService() {
 
     return userParsed;
   }
+
+  async function updateUser(userId: string, userInfo: UserUpdateInfo) {
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+    auditService.audit('alert-subscription.get', { userId });
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}`);
+    const response = await fetch(url, {
```

Comment:
I think this should use the new `instrumentedFetch()` function: <https://github.com/DTS-STN/canadian-dental-care-plan/blob/main/frontend/app/utils/fetch-utils.server.ts#L55>

Reply:
updated all fetchs